src/oemof/thermal/facades.py

```diff
@@ -0,0 +1,330 @@
+    storage_capacity: numeric
+        The total capacity of the storage (e.g. in MWh)
+    capacity: numeric
+        Maximum production capacity (e.g. in MW)
+    efficiency: numeric
+        Efficiency of charging and discharging process: Default: 1
+    storage_capacity_cost: numeric
+        Investment costs for the storage unit e.g in €/MWh-capacity
+    expandable: boolean
+        True, if capacity can be expanded within optimization. Default: False.
+    storage_capacity_potential: numeric
+        Potential of the investment for storage capacity in MWh
+    capacity_potential: numeric
+        Potential of the investment for capacity in MW
+    input_parameters: dict (optional)
+        Set parameters on the input edge of the storage (see oemof.solph for
+        more information on possible parameters)
+    ouput_parameters: dict (optional)
+        Set parameters on the output edge of the storage (see oemof.solph for
+        more information on possible parameters)
+    Intertemporal energy balance of the storage:
+    .. math::
+        x^{level}(t) =
+        x^{level}(t-1) \cdot (1 - c^{loss\_rate})
+        + \sqrt{c^{efficiency}(t)}  x^{flow, in}(t)
+        - \frac{x^{flow, out}(t)}{\sqrt{c^{efficiency}(t)}}
+        \qquad \forall t \in T
+    .. math::
+        x^{level}(0) = 0.5 \cdot c^{capacity}
+    The **expression** added to the cost minimizing objective funtion
+    for the operation is given as:
+    .. math::
+        x^{opex} = \sum_t (x^{flow, out}(t) \cdot c^{marginal\_cost}(t))
+
+    Examples
+    ---------
+    >>> from oemof import solph
+    >>> from oemof.thermal.facades import StratifiedThermalStorage
+    >>> heat_bus = solph.Bus(label='heat_bus')
+    >>> thermal_storage = StratifiedThermalStorage(
+    ...     label='thermal_storage',
+    ...     bus=heat_bus,
+    ...     diameter=10,
+    ...     height=10,
+    ...     temp_h=95,
+    ...     temp_c=60,
+    ...     temp_env=10,
+    ...     u_value=0.3,
+    ...     initial_storage_level=0.5,
+    ...     min_storage_level=0.05,
+    ...     max_storage_level=0.95
+    ...     capacity=1)
+    """
+
+    def __init__(self, *args, **kwargs):
+
+        super().__init__(
+            _facade_requires_=[
+                "bus", "diameter",
+                "temp_h", "temp_c", "temp_env",
+                "u_value"], *args, **kwargs
+        )
+
+        self.diameter = kwargs.get("diameter")
+
+        self.height = kwargs.get("height")
+
+        self.temp_h = kwargs.get("temp_h")
+
+        self.temp_c = kwargs.get("temp_c")
+
+        self.temp_env = kwargs.get("temp_env")
+
+        self.u_value = kwargs.get("u_value")
+
+        self.water_properties = {
+            'heat_capacity': kwargs.get("heat_capacity"), 'density': kwargs.get("density")
+        }
+
+        self.capacity = kwargs.get("capacity")
+
+        self.storage_capacity_cost = kwargs.get("storage_capacity_cost")
```

Comment:
```suggestion
        # self.storage_capacity_cost = kwargs.get("storage_capacity_cost")
```

There is no consequence if this argument is missing.